articles/quickstart/native/flutter/files/main.md

```diff
@@ -0,0 +1,68 @@
+---
+name: main_view.dart
+language: dart
+---
+
+```dart
+import 'package:auth0_flutter/auth0_flutter.dart';
+import 'package:flutter/material.dart';
+import 'profile_view.dart';
+
+class MainView extends StatefulWidget {
+  const MainView({Key? key}) : super(key: key);
+
+  @override
+  State<MainView> createState() => _MainViewState();
+}
+
+class _MainViewState extends State<MainView> {
+  bool _isLoggedIn = false;
```

Comment:
Let's get rid of the `_isLoggedIn` var, and use `if (_credentials == null)` on L35 instead. I've made the same change to the Swift snippets: https://github.com/auth0/docs/pull/10112